anker/tests/tests/sell_rewards.rs

```diff
@@ -0,0 +1,59 @@
+use anker::{error::AnkerError, token::MicroUst};
+use lido::token::Lamports;
+use solana_program::pubkey::Pubkey;
+use solana_program_test::tokio;
+use testlib::{anker_context::Context, assert_solido_error};
+
+const DEPOSIT_AMOUNT: u64 = 1_000_000_000; // 1e9 units
+
+#[tokio::test]
+async fn test_successful_sell_rewards() {
+    let mut context = Context::new_with_initialized_token_pool().await;
+    context.deposit(Lamports(DEPOSIT_AMOUNT)).await;
+    // Donate something to Solido's reserve so we can see some rewards.
+    context
+        .solido_context
+        .fund(
+            context.solido_context.reserve_address,
+            Lamports(DEPOSIT_AMOUNT),
+        )
+        .await;
+    // Update the exchange rate so we see some rewards.
+    context.solido_context.advance_to_normal_epoch(1);
+    context.solido_context.update_exchange_rate().await;
+
+    context.sell_rewards().await;
+
+    let ust_balance = context.get_ust_balance(context.ust_reserve).await;
+    // Exchange rate is 12 stSol : 13 Sol
```

Comment:
Where do these 12 and 13 come from? The pool gets initialized with 10 stSOL and 10 UST, right? And we add a 1 SOL deposit, and then 1 SOL of rewards ... shouldn’t we have 12 SOL and 11 stSOL? Where does the additional one come from?

Reply:
`Context::new()` also deposits 1 Sol.